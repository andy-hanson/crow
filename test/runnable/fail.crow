import
	bootstrap: fut, str, void

main fut nat(_ arr str) summon unsafe
	foo
	0 resolved

foo void() unsafe summon
	bar

bar void() unsafe summon
	fail: "FOO"
