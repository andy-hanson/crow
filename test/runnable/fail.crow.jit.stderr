uncaught exception: FOO
	at bar
	at foo
	at main_0
	at add__sfirst__stask__lambda0
	at call__sw__sctx_587
	at then__lambda0__lambda0
	at call__sw__sctx_587
	at catch_0__lambda0
	at call__sw__sctx_584
	at catch_1
	at catch_0
	at then__lambda0
	at call__sw__sctx_1110
	at invoke__saction
	at do__stask
	at thread__sfunction
